tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs

```diff
@@ -65,6 +85,77 @@ public void GraphicsExtension_Gd_IsInstalled(string imageTag)
             Assert.True((bool)((JValue)gdInfo.GetValue("PNG Support")).Value);
         }
 
+        [Theory]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        [InlineData("5.6")]
+        public async Task Check_If_Apache_Allows_Casing_In_PHP_File_Extension(string imageTag)
```

Comment:
ApacheServer_DoesNotServePhpFilesAsRawText_ForDifferentCasingOfFileExtension or something? (pivot here is to make sure Apache does not serve as raw text). Nitty thing as good test names can make code readable. You need not handle this comment in this PR.